src/components/DogDetail/DogDetail.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './DogDetail.css';
+
+export default function DogDetail({ dog, showDetail = false }) {
+  const getClassName = () => {
```

Comment:
I like how you have this set up and I see why you're doing it. However, and this is totally minor but, consider that the default value of border-radius is already 0 so you're telling all images to have a new border-radius, but then telling some of them to go back to default. Instead you could allow the default on the overall img element and then change to the custom radius on a class like 'small-dog' if that makes sense